src/Sentry.Maui/Internal/MauiEventsBinder.cs

```diff
@@ -1,364 +1,283 @@
 using Microsoft.Extensions.Options;
-using Sentry.Extensibility;
 
 namespace Sentry.Maui.Internal;
 
-internal class MauiEventsBinder : IMauiEventsBinder
+internal class MauiEventsBinder// : IMauiEventsBinder
 {
-    private readonly IHub _hub;
     private readonly SentryMauiOptions _options;
 
-    // https://develop.sentry.dev/sdk/event-payloads/breadcrumbs/#breadcrumb-types
-    // https://github.com/getsentry/sentry/blob/master/static/app/types/breadcrumbs.tsx
-    internal const string NavigationType = "navigation";
-    internal const string SystemType = "system";
-    internal const string UserType = "user";
-    internal const string HandlersCategory = "ui.handlers";
-    internal const string LifecycleCategory =  "ui.lifecycle";
-    internal const string NavigationCategory = "navigation";
-    internal const string RenderingCategory = "ui.rendering";
-    internal const string UserActionCategory = "ui.useraction";
-
-    // This list should contain all types that we have explicitly added handlers for their events.
-    // Any elements that are not in this list will have their events discovered by reflection.
-    internal static readonly HashSet<Type> ExplicitlyHandledTypes = new()
-    {
-        typeof(BindableObject),
-        typeof(Element),
-        typeof(VisualElement),
-        typeof(Application),
-        typeof(Window),
-        typeof(Shell),
-        typeof(Page),
-        typeof(Button),
-    };
-
     public MauiEventsBinder(IHub hub, IOptions<SentryMauiOptions> options)
     {
-        _hub = hub;
         _options = options.Value;
+
+        MauiEvents.Hub = hub;
+        MauiEvents.Options = _options;
```

Comment:
What 's the lifetime of this binder class? Having it mutate public static properties is really odd to me, I suggested above but I think it's best you put it all in 1 class (partial class cna help organize the code if u want to split in 2 files as it is now)